tests/integration/test_reads.py

```diff
@@ -342,7 +342,11 @@ def test_daft_nan_rewritten(catalog: Catalog) -> None:
 @pytest.mark.integration
 @pytest.mark.filterwarnings("ignore")
 @pytest.mark.parametrize("catalog", [pytest.lazy_fixture("session_catalog_hive"), pytest.lazy_fixture("session_catalog")])
-def test_bodo_nan(catalog: Catalog) -> None:
+def test_bodo_nan(catalog: Catalog, monkeypatch: pytest.MonkeyPatch) -> None:
+    # Avoid local Mac issues (see GH-2225)
```

Comment:
nit: 
idk if `GH-2225` will link to the issue
```suggestion
    # Avoid local Mac issues (see https://github.com/apache/iceberg-python/issues/2225)
```